assets/auto-test-case/dynamic/ui/rich-text-font-color.test.ts

```diff
@@ -0,0 +1,17 @@
+import { find } from 'cc';
+// @ts-ignore
+import { runScene, testCase, testClass } from 'db://automation-framework/runtime/test-framework.mjs';
+import { screenshot_custom } from '../common/utils';
+
+@runScene('rich-text-font-color')
+@testClass('RichTextFontColor')
```

Comment:
runScene is deprecated,  use @testClass('RichTextFontColor', 'rich-text-font-color')